quesma/quesma/search.go

```diff
@@ -146,6 +146,169 @@ type AsyncQuery struct {
 	startTime        time.Time
 }
 
+type executionPlanResult struct {
+	plan         *model.ExecutionPlan
+	err          error
+	responseBody []byte
+}
+
+func (q *QueryRunner) transformQueries(ctx context.Context, plan *model.ExecutionPlan, table *clickhouse.Table) {
+	var err error
+	plan.Queries, err = q.transformationPipeline.Transform(plan.Queries)
+	if err != nil {
+		logger.ErrorWithCtx(ctx).Msgf("error transforming queries: %v", err)
+	}
+
+	plan.Queries, err = registry.QueryTransformerFor(table.Name, q.cfg, q.schemaRegistry).Transform(plan.Queries)
+	if err != nil {
+		logger.ErrorWithCtx(ctx).Msgf("error transforming queries: %v", err)
+	}
+}
+
+func (q *QueryRunner) checkProperties(plan *model.ExecutionPlan, table *clickhouse.Table, queryTranslator IQueryTranslator, ctx context.Context) ([]byte, error) {
```

Comment:
Would add comment that this is potential legacy code to remove.

Reply:
Done